moped-editor/.env-cmdrc

```diff
@@ -85,7 +94,10 @@
     "REACT_APP_KNACK_DATA_TRACKER_PROJECT_ID_FIELD": "field_4133",
     "REACT_APP_KNACK_DATA_TRACKER_PROJECT_NAME_FIELD": "field_3857",
     "REACT_APP_KNACK_DATA_TRACKER_CURRENT_STATUS_FIELD": "field_4136",
-    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_3861"
+    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_3861",
+    "REACT_APP_KNACK_DATA_TRACKER_MOPED_URL": "field_4161",
+    "REACT_APP_KNACK_DATA_TRACKER_MOPED_LINK_LABEL": "field_4162",
+    "REACT_APP_KNACK_DATA_TRACKER_URL_BASE": "https://mobility.austin.gov/moped/projects/"
```

Comment:
will this eventually be https://moped.austinmobility.io/moped? I guess it doesnt matter if only devs are using the data tracker staging environment, right?

Reply:
Thank you for bringing up our dev/testing environment situation with regards to knack & moped. We got to discuss this briefly today in the dev sync, and I think we can mark this resolved until we decide it's the right time to build out a more robust development and testing setup for this type of thing. Thanks again!